library/examples/hate_speech.py

```diff
@@ -166,3 +171,40 @@ def create_nlp_huggingface_pipeline(autocorrect: bool) -> Pipeline:
     "ensemble_hf_statistic_sklearn",
     [nlp_sklearn, text_statistics_pipeline, huggingface_baseline],
 )
+
+
+data_emoji = transform_dataset(load_dataset("tweet_eval", "emoji"), preprocess_config)
+
+data_tweet_eval_hate_speech = transform_dataset(
+    load_dataset("tweet_eval", "hate"), preprocess_config
+)
+data_hatecheck = transform_hatecheck_dataset(
+    load_dataset("Paul/hatecheck"), preprocess_config
+)
+
+tweeteval_hate_speech_run_configs = [
+    RunConfig(
+        run_name="hate-speech-detection",
+        dataset=data_tweet_eval_hate_speech[0],
+        train=True,
+    ),
+    RunConfig(
+        run_name="hate-speech-detection",
+        dataset=data_tweet_eval_hate_speech[1],
+        train=False,
+    ),
+]
+
+
+cross_dataset_run_configs = [
+    # RunConfig(
+    #     run_name="hate-speech-detection-cross-val",
+    #     dataset=data_tweet_eval_hate_speech[0],
+    #     train=True,
+    # ),
+    RunConfig(
+        run_name="hate-speech-detection-cross-val",
```

Comment:
this one has a different name as we can't compare it to evaluation of the tweeteval hate speech dataset.